src/core/github.js

```diff
@@ -50,19 +58,23 @@ export async function run(conf) {
   try {
     ghURL = new URL(tempURL, "https://github.com");
   } catch {
-    pub("error", `\`respecConf.github\` is not a valid URL? (${ghURL})`);
+    const msg = `\`respecConf.github\` is not a valid URL? (${ghURL})`;
+    pub("error", msg);
+    apiURLRejector(new Error(msg));
     return;
   }
   if (ghURL.origin !== "https://github.com") {
     const msg = `\`respecConf.github\` must be HTTPS and pointing to GitHub. (${ghURL})`;
     pub("error", msg);
+    apiURLRejector(new Error(msg));
```

Comment:
This doesn't seem to be a scalable approach :disappointed:

Reply:
Can you help me understand your thinking? I was thinking that in the dependent module this prevents the module from gracefully proceeding.

Reply:
By salability, I meant: if we want to make more things available outside, we need to export them - each having its own promise and having a reject call - which becomes cumbersome.
For this one, I decided to export an object, instead of adding a new promise for each thing.